docs/guides/tracks.md

```diff
@@ -52,157 +58,114 @@ The default attribute can be used to have a track default to showing. Otherwise
-        <tr><th>to<th><td>Tonga</td></tr>
-        <tr><th>ts<th><td>Tsonga</td></tr>
-        <tr><th>tr<th><td>Turkish</td></tr>
-        <tr><th>tk<th><td>Turkmen</td></tr>
-        <tr><th>tw<th><td>Twi</td></tr>
-        <tr><th>ug<th><td>Uighur</td></tr>
-        <tr><th>uk<th><td>Ukrainian</td></tr>
-        <tr><th>ur<th><td>Urdu</td></tr>
-        <tr><th>uz<th><td>Uzbek</td></tr>
-        <tr><th>vi<th><td>Vietnamese</td></tr>
-        <tr><th>vo<th><td>Volapük</td></tr>
-        <tr><th>wa<th><td>Wallon</td></tr>
-        <tr><th>cy<th><td>Welsh</td></tr>
-        <tr><th>wo<th><td>Wolof</td></tr>
-        <tr><th>xh<th><td>Xhosa</td></tr>
-        <tr><th>yi<th><td>Yiddish</td></tr>
-        <tr><th>yo<th><td>Yoruba</td></tr>
-        <tr><th>zu<th><td>Zulu</td></tr>
-      </table>
-  </tr>
-</table>
+The two-letter code (valid BCP 47 language tag) for the language of the text track, for example "en" for English. A list of language codes is [available here](languages.md#language-codes).
+
+## API
+
+### `player.textTracks() -> TextTrackList`
+This is the main interface into the text tracks of the player.
+It return a TextTrackList which lists all the tracks on the player.
+
+### `player.remoteTextTracks() -> TextTrackList`
+This is a helper method to get a list of all the tracks that were created from `track` elements or that were added to the player by the `addRemoteTextTrack` method. All these tracks are removeable from the player, where-as not all tracks from `player.textTracks()` are necessarily removeable.
+
+### `player.remoteTextTrackEls() -> HTMLTrackElementList`
+Another helper method, this is a list of all the `track` elements associates with the player. Both emulated or otherwise.
+
+### `player.addTextTrack(String kind, [String label [, String language]]) -> TextTrack`
+This is based on the [w3c spec API](http://www.w3.org/html/wg/drafts/html/master/embedded-content-0.html#dom-media-addtexttrack) and when given a kind and an optional label and language, will create a new text track for you to use.
+This method is intended for purely programmatic usage of tracks and has one important limitation:
+tracks created using this method *cannot* be removed. The native `addTextTrack` does not have a corresponding `removeTextTrack`, so, we actually discourage the usage of this method.
+
+### `player.addRemoteTextTrack(Object options) -> HTMLTrackElement`
+This function takes an options object that looks pretty similar to the track element and returns a HTMLTrackElement.
+This object has a `track` property on it which is the actual TextTrack object.
+This `TextTrack` object is equivalent to the one that can be returned from `player.addTextTrack` with the added bonus that it can be removed from the player.
+Internally, video.js will either add a `<track>` element for you, or emulate that depending on whether native text tracks are supported or not.
+The options available are:
+* `kind`
+* `label`
+* `language` (also `srclang`)
```

Comment:
Oh, I was confusing `language` with `lang` (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang); that's the thing which is distinct from `srclang`.